applications/app/AppLoader.scala

```diff
@@ -32,21 +32,18 @@ trait ApplicationsServices {
   lazy val contentApiClient = wire[ContentApiClient]
   lazy val siteMapJob = wire[SiteMapJob]
   lazy val sectionsLookUp = wire[SectionsLookUp]
+  lazy val ophanApi = wire[OphanApi]
 }
 
-trait Controllers extends ApplicationsControllers {
-  self: FrontendComponents with ApplicationsServices =>
-  def wsClient: WSClient
+
+trait AppComponents extends FrontendComponents with ApplicationsControllers with ApplicationsServices {
+
```

Comment:
Unrelated again: Removing the unnecessary traits in `AppLoader`. I have done that a few time in this PR